extension/popup/popupView.jsx

```diff
@@ -350,6 +350,29 @@ const ListeningContent = ({
         <VoiceInput suggestions={suggestions} onClickLexicon={onClickLexicon} />
         <TypingInput onInputStarted={onInputStarted} />
       </div>
+      <div className="popup-tips">
```

Comment:
So, from the comment https://github.com/mozilla/firefox-voice/issues/1188#issuecomment-597930821 we only want to show this in developer builds.

I was going to explain how to check and realized it was too hard, so I made a quick change: https://github.com/mozilla/firefox-voice/pull/1307/files

Now you can just check `buildSettings.inDevelopment` and only show these tips when that is true.

Reply:
Thanks! and Duly noted. I will be closing this PR cause it is one of the orphaned PRs and work from a new one.